src/tests/ftest/deployment/agent_failure.yaml

```diff
@@ -0,0 +1,70 @@
+hosts:
+    test_servers:
+        - server-A
+        - server-B
+        - server-C
+        - server-D
+        - server-E
+        - server-F
+        - server-G
+    test_clients:
+        - client-A
+
+timeout: 300
+
+setup:
+    start_agents_once: False
+
+daos_server:
+  pattern_timeout: 60
+
+server_config:
+    name: daos_server
+    engines_per_host: 2
+    servers:
+        0:
+            pinned_numa_node: 0
+            nr_xs_helpers: 1
+            fabric_iface: ib0
+            fabric_iface_port: 31317
+            log_file: daos_server0.log
+            bdev_class: nvme
+            bdev_list: ["0000:00:00.0"]
+            scm_class: dcpm
+            scm_list: ["/dev/pmem0"]
+            scm_mount: /mnt/daos0
+        1:
+            pinned_numa_node: 1
+            nr_xs_helpers: 1
+            fabric_iface: ib1
+            fabric_iface_port: 31417
+            log_file: daos_server1.log
+            bdev_class: nvme
+            bdev_list: ["0000:00:00.1"]
+            scm_class: dcpm
+            scm_list: ["/dev/pmem1"]
+            scm_mount: /mnt/daos1
+
+pool:
+    size: 100%
+    control_method: dmg
+
+container:
+    type: POSIX
+    control_method: daos
+
+ior:
+    client_processes:
+        np: 4
+    iorflags:
+        flags: "-k -D 300 -v -w -W"
+        api: DFS
+        # This test doesn't work if the IOR runs too fast. In that case, increase
+        # block_size and/or decrease transfer_size. To decrease transfer_size, divide it
+        # by 2, 4, 8, etc.
+        transfer_size: '256K'
+        block_size: '20G'
+        write_x: 4
+        read_x: 2
```

Comment:
1. I think write_x and read_x can be removed (unused).
2. `-D 300` sets a stonewall of 300s. Is this intentional? I think we should either (A) remove the stonewall, or (B) increase the block size high enough that IOR runs for 300s (5 minutes)

Reply:
Thanks for the comment. I'll remove `write_x` and `read_x`.
I haven't put too much thought into the stonewall value. This test will run in TDS, so setting these values now would be tricky. We want to generate a lot of data, but we don't want it to run too long. We don't want it to run too fast either because we'll end up stopping the agent after the process ends. I can change some of the values, but I'll probably end up adjusting them when I bring this test to TDS.

Reply:
I see. That's good to know. Thanks! (Although it's still tricky because we need to make sure the IO takes longer than `sw_deadline`, but we need to take the environment such as disk space, network bandwidth, etc. into account.)

Reply:
Yep! But at least with low stonewall on "real" hardware, it's easier. For example, with a 30s stonewall, if a server node has 2 drives, spec'ed at 20G/s, then the max IO size is `20G/s * 2 * 30s = 1200G ~ 1.2T` per server node. Then, the stonewall can be adjusted to account for smaller or larger drives. In fact, [this IOR test](https://github.com/daos-stack/daos/blob/master/src/tests/ftest/ior/intercept_multi_client.yaml) uses stonewall 60s with 32 PPN on HW Medium.
The benefit to this approach is that it's easier to scale with more server/client nodes.
But this is just one method. For example, Michael H. likes to calculate block size, etc. based on the hardware. This is less flexible, but has some other pros.

Anyway, no need to change at this time :)